app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -127,174 +125,179 @@ private fun DetailsContent(
+            val keyboardOptions = KeyboardOptions(KeyboardCapitalization.Words, true, KeyboardType.Text, ImeAction.Next)
+            val keyboardController = LocalSoftwareKeyboardController.current
+            val focusRequesterFirstName = remember { FocusRequester() }
+            val focusRequesterLastName = remember { FocusRequester() }
+            val focusRequesterTeamName = remember { FocusRequester() }
+
+            LazyColumn(
+                state = listState,
                 horizontalAlignment = Alignment.CenterHorizontally,
                 verticalArrangement = Arrangement.Top,
                 modifier = Modifier
-                    .weight(weight = 1f, fill = true)
-                    .verticalScroll(scrollState)
+                    .weight(weight = 1f)
+                    .imeNestedScroll()
             ) {
-                Text(
-                    text = stringResource(R.string.create_personal_account_details_text),
-                    style = MaterialTheme.wireTypography.body01,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(
-                            horizontal = MaterialTheme.wireDimensions.spacing16x,
-                            vertical = MaterialTheme.wireDimensions.spacing24x
+                item {
+                    Text(
+                        text = stringResource(R.string.create_personal_account_details_text),
+                        style = MaterialTheme.wireTypography.body01,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(
+                                horizontal = MaterialTheme.wireDimensions.spacing16x,
+                                vertical = MaterialTheme.wireDimensions.spacing24x
+                            )
+                    )
+                }
+
+                item {
+                    WireTextField(
+                        value = state.firstName,
+                        onValueChange = onFirstNameChange,
+                        placeholderText = stringResource(R.string.create_account_details_first_name_placeholder),
+                        labelText = stringResource(R.string.create_account_details_first_name_label),
+                        labelMandatoryIcon = true,
+                        state = WireTextFieldState.Default,
+                        keyboardOptions = keyboardOptions,
+                        modifier = Modifier
+                            .padding(
+                                start = MaterialTheme.wireDimensions.spacing16x,
+                                end = MaterialTheme.wireDimensions.spacing16x,
+                                bottom = MaterialTheme.wireDimensions.spacing16x
+                            )
+                            .focusRequester(focusRequesterFirstName)
+                            .testTag("firstName"),
+                    )
```

Comment:
First element `firstName` doesn't have `onTap` implemented, 
```
onTap = {
                            coroutineScope.launch {
                                keyboardController?.show()
                                focusRequesterFirstName.requestFocus()
                                listState.animateScrollToItem(0)
                            }
                        }
```
so when we click back on first input to edit it for instance, it won't change focus and scroll to that item?

Reply:
Sure, you're right, scrolling up shouldn't be a problem as the user should be already aware of the inputs that are there